newswires/client/src/WireItemList.tsx

```diff
@@ -162,21 +184,32 @@ const WirePreviewCard = ({
 	highlight: string | undefined;
 	selected: boolean;
 	isFromRefresh: boolean;
+	view: string;
+	previousItemId: string | undefined;
 }) => {
 	const { viewedItemIds } = useSearch();
 	const { showSecondaryFeedContent } = useUserSettings();
 
 	const ref = useRef<HTMLDivElement>(null);
+	const isSmallScreen = useIsWithinBreakpoints(['xs', 's']);
 
 	useEffect(() => {
 		if (selected && ref.current) {
-			ref.current.scrollIntoView({
-				behavior: 'smooth',
-				block: 'nearest',
-			});
+			scrollElementIntoView(ref.current);
 		}
 	}, [selected]);
 
+	useEffect(() => {
+		if (
+			isSmallScreen &&
```

Comment:
is this meant to target the case of the regular mobile view only? if so, we need to add `!isPoppedOut` to the list of conditions

my feeling is that we should add a comment to this useEffect hook to explain the intention, as there are quite a few conditions in play so it's not immediately obvious to a reader unless they work through exactly what conditions would trigger the scrolling behaviour